cms/static/js/certificates/models/certificate.js

```diff
@@ -0,0 +1,81 @@
+// Backbone.js Application Model: Certificate
+
+define([
+    'backbone', 'js/certificates/models/signatory', 'js/certificates/collections/signatories', 'underscore', 'underscore.string', 'gettext', 'backbone-relational', 'backbone.associations', 'coffee/src/main'
+],
+function(Backbone, Signatory, SignatoryCollection, _, str, gettext) {
+    'use strict';
+    _.str = str;
+    var Certificate = Backbone.RelationalModel.extend({
+        idAttribute: "id",
+        defaults: {
+            name: 'Name of the certificate',
```

Comment:
If these defaults are not going away (I believe they are!), then they need to be internationalized.